NPE when clicking on "back" button

I use version 4.0 from the Play store.

I was opening an issue from a Github email ("view it on Github") and then hit the "<-" button.

This is the exception I got:

```
E/AndroidRuntime: FATAL EXCEPTION: main
Process: com.gh4a, PID: 22886
java.lang.RuntimeException: Unable to start activity ComponentInfo{com.gh4a/com.gh4a.activities.IssueListActivity}: java.lang.NullPointerException: Attempt to invoke virtual method 'void android.view.View.setScaleX(float)' on a null object reference
    at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2426)
    at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2490)
    at android.app.ActivityThread.-wrap11(ActivityThread.java)
    at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1354)
    at android.os.Handler.dispatchMessage(Handler.java:102)
    at android.os.Looper.loop(Looper.java:148)
    at android.app.ActivityThread.main(ActivityThread.java:5443)
    at java.lang.reflect.Method.invoke(Native Method)
    at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:728)
    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:618)
 Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'void android.view.View.setScaleX(float)' on a null object reference
    at android.support.v4.view.ViewCompatHC.setScaleX(ViewCompatHC.java:126)
    at android.support.v4.view.ViewCompat$HCViewCompatImpl.setScaleX(ViewCompat.java:1177)
    at android.support.v4.view.ViewCompat.setScaleX(ViewCompat.java:2613)
    at com.gh4a.activities.IssueListActivity.onPageMoved(IssueListActivity.java:270)
    at com.gh4a.BasePagerActivity.onPostCreate(BasePagerActivity.java:53)
    at android.app.Instrumentation.callActivityOnPostCreate(Instrumentation.java:1211)
    at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2408)
    at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2490) 
    at android.app.ActivityThread.-wrap11(ActivityThread.java) 
    at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1354) 
    at android.os.Handler.dispatchMessage(Handler.java:102) 
    at android.os.Looper.loop(Looper.java:148) 
```

I can reproduce it 100%. I'm not authenticated with Github within the app.

Oh, forgot to mention: I'm using Android 6.0.1.

--- a/src/com/gh4a/activities/IssueListActivity.java
+++ b/src/com/gh4a/activities/IssueListActivity.java
@@ -265,7 +265,7 @@ public class IssueListActivity extends BasePagerActivity implements
     @Override
     protected void onPageMoved(int position, float fraction) {
         super.onPageMoved(position, fraction);
-        if (!mSearchMode) {
+        if (!mSearchMode && mCreateFab != null) {
             float openFraction = 1 - position - fraction;
             ViewCompat.setScaleX(mCreateFab, openFraction);
             ViewCompat.setScaleY(mCreateFab, openFraction);

